comparisontool/search_indexes.py

```diff
@@ -1,21 +0,0 @@
-from haystack import indexes
```

Comment:
Should [the index tests](https://github.com/cfpb/django-college-costs-comparison/blob/main/comparisontool/tests/test_search_indexes.py) also be deleted?

Reply:
Although things like unused templates might not be worth messing with, because the whole repo will be mothballed soonish.

Reply:
@chosak Unneeded templates gone.